### DISCUSSION TITLE
Welcome to ConcePTIONAlgorithmPregnancies Discussions!

### DISCUSSION THREAD

**Discussion by RosaMGini** [ARS-toscana/ConcePTIONAlgorithmPregnancies#55] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by EmmyManders** [comment]
@GiorgioLimoncella @RosaMGini  
For Pfizer 1021 and other projects the  ConcePTION Pregnancy Algorithm is used in DEAPs. I was wondering if you could give me more information on how start and end_pregnancy_dates can be imputed. What are the rules of imputation? Is only the day imputed or also month? 
Thank you. Emmy

**Comment by GiorgioLimoncella** [comment]
Dear Emmy, 

The algorithm searches each data source for all records that imply a woman is pregnant, and then determines pregnancy start and end dates by reconciling these records according to a hierarchy of "reliability". The complete set of rules is described in the ConcePTION Deliverable [link](https://zenodo.org/records/13860418) (table 2.2 shows the hierarchy of records).

Several DEAP-specific rules were applied for the imputations, but in short:

- Start of pregnancy: when not directly available from the records, it is imputed using a predictive model (Random Forest). Specifically, the model estimates the gestational age (in days).

- End of pregnancy: when not directly available, it is not predicted. Instead, each DEAP defined its approach between two options:  (a) start date + 280 days, or (b) the date of the last pregnancy-related record (in DEAPs where pregnancies without an explicit end are considered to reflect an unfavorable outcome).

Finally, each pregnancy is assigned a "color quality" that indicates whether the start date, the end date, or both dates have been imputed.

We have submitted a manuscript describing the application of the algorithm to 10 DEAPs, and the preprint is available at this [link](https://www.authorea.com/users/943555/articles/1313611-identifying-pregnancies-in-population-based-data-sources-development-and-application-of-the-conception-pregnancy-algorithm?commit=dc1b766f4701f1bce355cfa0fa5e43ed2444b306).

Please don’t hesitate to ask if you have any specific questions!

Best regards, 
Giorgio

**Reply by EmmyManders** [reply]
Thank you @GiorgioLimoncella
I'll read the preprint first and come back to you if I have more questions.

**Comment by EmmyManders** [comment]
@GiorgioLimoncella 
What does the algorithm do when it detects overlapping pregnancy episodes? Does that depend on the quality of one or both of the pregnancies? Is a pregnancy excluded? 
I'm asking as we see this with CPRD who have developed their own algorithm. We want to push them to handle those the same as the ConceptionAlgorithm. 
Thank you! KR Emmy

**Reply by GiorgioLimoncella** [reply]
@EmmyManders 
It depends on the quality of both overlapping pregnancies. It’s fully described in box 2.1 of the deliverable.
In about half an hour (3pm) we have a meeting with the pregnancy algorithm task force. If you’d like to join just write me at giorgio.limoncella@ars.toscana.it and I'll share the meeting link. We can quickly go through your questions!

Best, 
Giorgio
